src/Symfony/Component/HttpKernel/Tests/KernelTest.php

```diff
@@ -36,6 +36,10 @@ public static function tearDownAfterClass()
         $fs->remove(__DIR__.'/Fixtures/var');
     }
 
+    /**
+     * @group legacy
```

Comment:
Adding the test to the legacy group means that it will be removed in `master`. This is probably not what we want here. I suggest removing the `getContainer()` call and add a new legacy test instead that tries to get a container from the kernel before it was booted.